pkg/identity/certificated.go

```diff
@@ -256,19 +264,20 @@ func Certificated(idConfig *config.IdentityConfig, stopChan <-chan struct{}) (er
 			log.Infof("Attempting to request renewed x509 certificate to identity provider[%s]...", idConfig.ProviderService)
 			err, forceInitIdentity, forceInitKeyPEM = identityProvisioningRequest(true)
 			if err != nil {
+				// TODO: Maybe Warning instead?
 				log.Errorf("Failed to retrieve renewed x509 certificate from identity provider: %s", err.Error())
```

Comment:
@mlajkim I think we should have a conclusion for this.
```suggestion
				// TODO: Maybe Warning instead?
				log.Errorf("Failed to retrieve renewed x509 certificate from identity provider: %s", err.Error())
```

Reply:
@ctyano Since the FORCE_INIT process is not a must-pass process, it might be a bit weird to log ERROR when the flow continues and function properly.

But, if it works as a reminder that the pod is living with the backed up secret, then Error still makes sense.

Reply:
> Since the FORCE_INIT process is not a must-pass process, it might be a bit weird to log ERROR when the flow continues and function properly.

@mlajkim I totally agree. We can actually say the same thing for `identityProvisioningRequest` process but, at least for now, we should be printing ERROR only when there will be any affect on the output for the workload.
As you mentioned, if the flow continues and function properly, there should be only WARNING to be printed.

Reply:
```suggestion
				log.Warnf("Failed to retrieve renewed x509 certificate from identity provider: %s, keep using backup from kubernetes secret", err.Error())
```
WARNING +1

Reply:
```suggestion
				log.Warnf("Failed to retrieve renewed x509 certificate from identity provider: %s, continuing with the backup certificate in k8s secret", err.Error())
```
WARNING +1

Reply:
Created an issue for this
https://github.com/AthenZ/k8s-athenz-sia/issues/63

Reply:
`in k8s secret` => `from kubernetes secret`, consistency with other logs